tests/e2e/streams/kafka_streams_tests.py

```diff
@@ -439,27 +372,39 @@ def test_info_procedure(kafka_topics, connection):
         f"CONSUMER_GROUP {consumer_group} "
         f"BOOTSTRAP_SERVERS '{local}' "
         f"CONFIGS {configs} "
-        f"CREDENTIALS {credentials}"
+        f"CREDENTIALS {credentials}",
     )
 
-    stream_info = common.execute_and_fetch_all(
-        cursor, f"CALL mg.kafka_stream_info('{stream_name}') YIELD *")
+    stream_info = common.execute_and_fetch_all(cursor, f"CALL mg.kafka_stream_info('{stream_name}') YIELD *")
 
-    reducted_credentials = {key: "<REDUCTED>" for
-                            key in credentials.keys()}
+    reducted_credentials = {key: "<REDUCTED>" for key in credentials.keys()}
 
-    expected_stream_info = [
-        (local, configs, consumer_group, reducted_credentials, kafka_topics)]
+    expected_stream_info = [(local, configs, consumer_group, reducted_credentials, kafka_topics)]
     common.validate_info(stream_info, expected_stream_info)
 
-@pytest.mark.parametrize("transformation",TRANSFORMATIONS_TO_CHECK_C)
+
+@pytest.mark.parametrize("transformation", TRANSFORMATIONS_TO_CHECK_C)
 def test_load_c_transformations(connection, transformation):
     cursor = connection.cursor()
-    query = "CALL mg.transformations() YIELD * WITH name WHERE name STARTS WITH 'c_transformations." + transformation + "' RETURN name" 
-    result = common.execute_and_fetch_all(
-                 cursor, query)
+    query = "CALL mg.transformations() YIELD * WITH name WHERE name STARTS WITH '" + transformation + "' RETURN name"
+    result = common.execute_and_fetch_all(cursor, query)
     assert len(result) == 1
-    assert result[0][0] == "c_transformations." + transformation
-    
+    assert result[0][0] == transformation
+
+
+@pytest.mark.parametrize("transformation", TRANSFORMATIONS_TO_CHECK_PY)
+def test_start_stream_with_batch_limit(kafka_producer, kafka_topics, connection, transformation):
```

Comment:
```suggestion
def test_start_stream_with_batch_limit(kafka_producer, kafka_topics, connection):
```

Does this test need to be parameterized based on the transformation? I think different transformations shouldn't make a difference in the test case.